docs/ops/deployment/yarn_setup.md

```diff
@@ -132,7 +132,7 @@ Stop the YARN session by stopping the unix process (using CTRL+C) or by entering
 
 Flink on YARN will only start all requested containers if enough resources are available on the cluster. Most YARN schedulers account for the requested memory of the containers,
 some account also for the number of vcores. By default, the number of vcores is equal to the processing slots (`-s`) argument. The `yarn.containers.vcores` allows overwriting the
-number of vcores with a custom value.
+number of vcores with a custom value. In order for this parameter to work you should enable CPU scheduling in your cluster, see more [here]({{ site.baseurl }}/ops/config.html#yarn-containers-vcores)
```

Comment:
You should, however, never link on "here" - maybe put the link on "enable CPU scheduling" instead?